a
No exception
b
Uncaught exception Raw_backtrace.Error("b")
Raised at file "raw_backtrace.ml", line 7, characters 16-32
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 11, characters 4-11
Re-raised at file "raw_backtrace.ml", line 13, characters 62-71
Called from file "raw_backtrace.ml", line 18, characters 11-23
Uncaught exception Raw_backtrace.Error("c")
Raised at file "raw_backtrace.ml", line 14, characters 20-37
Called from file "raw_backtrace.ml", line 18, characters 11-23
Uncaught exception Raw_backtrace.Error("d")
Raised at file "raw_backtrace.ml", line 7, characters 16-32
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 7, characters 42-53
Called from file "raw_backtrace.ml", line 11, characters 4-11
Called from file "raw_backtrace.ml", line 18, characters 11-23
Uncaught exception Invalid_argument("index out of bounds")
Raised by primitive operation at file "raw_backtrace.ml", line 18, characters 14-22
